src/scripts/main.js

```diff
@@ -1,3 +1,64 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const listSalary = document.querySelector('tbody').children;
+
+const TRIGERNAME_SALARY = { name: 'Salary', index: 3 };
+const TRIGERNAME_AGE = { name: 'Age', index: 2 };
+const TRIGERNAME_POSITION = { name: 'Position', index: 1 };
+const TRIGERNAME_NAME = { name: 'Name', index: 0 };
```

Comment:
```suggestion
const SOME_COOL_NAME = {'Salary': 3, 'Age': 2, ......};
```
looks like you can simplify it